pybricks/pybricks.c

```diff
@@ -133,10 +136,25 @@ void pb_package_pybricks_init(bool import_all) {
 // REVISIT: move these to object finalizers if we enable finalizers in the GC
 void pb_package_pybricks_deinit(void) {
     #if PYBRICKS_PY_COMMON_BLE
-    pb_type_BLE_cleanup();
+    pb_type_ble_start_cleanup();
     #endif
     // Disconnect from remote.
     #if PYBRICKS_PY_PUPDEVICES
-    pb_type_Remote_cleanup();
+    pb_type_lwp3device_start_cleanup();
     #endif // PYBRICKS_PY_PUPDEVICES
+
+    #if PYBRICKS_PY_COMMON_BLE && PYBRICKS_PY_PUPDEVICES
```

Comment:
Should this be `||` instead of `&&`?

Reply:
Thanks. I'll probably just have to be specific and add `PYBRICKS_PY_PUPDEVICES_REMOTE` to account for the virtualhub which has pupdevices but no remote, and then we can use `||`.